tests/PySys/tedge_mapper_c8y/tedge_mapper_c8y_alarms/run.py

```diff
@@ -25,6 +25,10 @@ def execute(self):
         # Solution: only inject the variables we really need
         environ={ 'HOME':os.environ.get('HOME')}
 
+        # TODO to wait for 5s until everythig is settled in the background is probably to
+        # long -> test and reduce
+        time.sleep(5)
```

Comment:
May I ask why this is necessary in the first place? What needs to settle in the background exactly?